fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -1419,7 +1419,12 @@ public void handleDMLStmt(ExecPlan execPlan, DmlStmt stmt) throws Exception {
         String dbName = stmt.getTableName().getDb();
         String tableName = stmt.getTableName().getTbl();
         Database database = GlobalStateMgr.getCurrentState().getMetadataMgr().getDb(catalogName, dbName);
-        Table targetTable = GlobalStateMgr.getCurrentState().getMetadataMgr().getTable(catalogName, dbName, tableName);
```

Comment:
can we keep this unchanged?

Reply:
If the target table is specified, we should use it, because the target table has the full metadata for OLAPExternalTable.